pkg/Microsoft.ML/Microsoft.ML.nupkgproj

```diff
@@ -11,7 +11,6 @@
 
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="$(SystemReflectionEmitLightweightPackageVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Dataflow" Version="$(SystemThreadingTasksDataflowPackageVersion)" />
```

Comment:
You also need to add `<PackageReference Include="System.Threading.Channels" Version="$(SystemThreadingChannelsPackageVersion)" />` here.